{"name": "test_login_success", "status": "broken", "statusDetails": {"message": "TypeError: unhashable type: 'dict'", "trace": "@pytest.fixture(scope='session',autouse=True)\n    def session_fixtrue10():\n        path='/api/learning/v3/judge'\n>       date4=date1[date1]\nE       TypeError: unhashable type: 'dict'\n\nconftest.py:80: TypeError"}, "start": 1586521603798, "stop": 1586521603798, "uuid": "ebd2e9e3-f334-45f0-b2d3-af851a69c064", "historyId": "b40641d31f013d71b08e9edead2f906b", "testCaseId": "40328625064c5c8b47b3abdd23fe1487", "fullName": "testcases.test_login.Test_login#test_login_success", "labels": [{"name": "story", "value": "登录成功"}, {"name": "feature", "value": "登录成功"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login"}, {"name": "host", "value": "DESKTOP-938U3RR"}, {"name": "thread", "value": "11124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_login"}]}